lms/views/api/blackboard/files.py

```diff
@@ -14,21 +28,32 @@ def __init__(self, request):
     @view_config(request_method="GET", route_name="blackboard_api.courses.files.list")
     def list_files(self):
         """Return the list of files in the given course."""
-        return self.blackboard_api_client.list_files(
-            self.request.matchdict["course_id"]
+        course_id = self.request.matchdict["course_id"]
+        response = self.blackboard_api_client.request(
+            "GET", f"courses/uuid:{course_id}/resources"
         )
+        return BlackboardListFilesSchema(response).parse()
```

Comment:
The `list_files()` view gets a bit longer but it's zero logic: no `if`'s or loops or anything